client/src/components/check-out-item/check-out-item.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
checkout item component in the checkout page. user able to add and delete items. The total of the cart is also displayed here